docs/analyzer/user_guide.md

```diff
@@ -343,6 +343,32 @@ Example:
 CodeChecker log -o ../codechecker_myProject_build.log -b "make -j2"
 ```
 
+### Change user inside the build command
+If we change user inside the build command of the CodeChecker log command
+before the actual compiler invocation, the compilation database will be empty:
+
+```sh
+CodeChecker log -o compile_commands.json -b "su myuser  -c 'g++ main.cpp -o /dev/null'"
+```
+
+The problem at here that the compilation database file and the lock file will
+be created with the user who runs the CodeChecker log command and only this
+user will have permission to read/write these files. So after changing to a
+different user, this user will not have permissions on these files.
```

Comment:
Minor comment: this sentence seems redundant to me, because previous one also talks about the lack of permission.